pkg/providers/aws/cluster_network_provider_test.go

```diff
@@ -704,6 +797,11 @@ func TestNetworkProvider_CreateNetwork(t *testing.T) {
 					ec2Client.subnets = buildStandaloneVPCAssociatedSubnets(defaultValidSubnetMaskOneA, defaultValidSubnetMaskOneB)
 					ec2Client.azs = buildSortedStandaloneAZs()
 					ec2Client.firstSubnet = buildSubnet(defaultStandaloneVpcId, defaultSubnetIdOne, defaultAzIdOne, defaultValidSubnetMaskOneA)
+					ec2Client.describeSubnetsFn = func(input *ec2.DescribeSubnetsInput) (*ec2.DescribeSubnetsOutput, error) {
+						return &ec2.DescribeSubnetsOutput{
+							Subnets: buildValidBundleSubnets(),
```

Comment:
I think this might be the same, as we are testing a non tagged standalone vpc.

Reply:
replaced with standalone